apps/dashboard/src/components/workflow-editor/test-workflow/test-workflow-logs-sidebar.tsx

```diff
@@ -61,36 +67,43 @@ export const TestWorkflowLogsSidebar = ({ transactionId, workflow }: TestWorkflo
         </div>
       ) : activityId ? (
         <>
-          <ActivityPanel
-            activityId={activityId}
-            onActivitySelect={setParentActivityId}
-            headerClassName="h-[49px]"
-            overviewHeaderClassName="border-t-0"
-          />
-          {!workflow?.lastTriggeredAt && (
-            <div className="p-3">
-              <div className="border-stroke-soft bg-bg-weak rounded-8 flex items-center justify-between gap-3 border p-3 py-2">
-                <div className="flex items-center gap-3">
-                  <div className="bg-success-100 flex size-6 items-center justify-center rounded-full">
-                    <RiCheckboxCircleFill className="text-success size-5" />
-                  </div>
-                  <div>
-                    <div className="text-success text-label-xs">You have triggered the workflow!</div>
-                    <div className="text-text-sub text-label-xs">Now integrate the workflow in your application.</div>
+          <ActivityPanel>
+            {isPending ? (
+              <ActivitySkeleton />
+            ) : error || !activity ? (
+              <ActivityLoadFailure />
+            ) : (
+              <>
+                <ActivityHeader title={activity.template?.name} className="h-[49px] border-t-0" />
+                <ActivityOverview activity={activity} />
+                <ActivityLogs activity={activity} onActivitySelect={setParentActivityId} />
+              </>
+            )}
+            {!workflow?.lastTriggeredAt && (
+              <div className="border-t border-neutral-100 p-3">
```

Comment:
this part was moved under the panel to fix the overflow issue with "integrate workflow" banner